### ISSUE TITLE
Internationalisation updates (normative)

### ISSUE THREAD

**Issue by alastc** [w3c/wcag#4263]
There is an issue about how some WCAG 2.0/2.1 criteria apply across languages/scripts.

There is a [long-ish list on the i18n repo](https://github.com/w3c/i18n-activity/issues?q=label%3Awg%3Aag%20sort%3Acreated-asc), and a [recent issue](https://github.com/w3c/i18n-discuss/issues/46) for a reasonable overview.

This issue is to provide a place to track potential normative updates to WCAG 2.2. These would be [class 3 updates](https://www.w3.org/policies/process/#correction-classes), so will need to go through full process. (And then we'd have to work out how to publish in terms of versioning etc.)

As an example, [murata2makoto](https://github.com/murata2makoto) suggested:

<hr>

Here is my proposed rewrite of [1.4.12 Text Spacing](https://www.w3.org/WAI/WCAG22/quickref/#text-spacing).

> Line height (line spacing) to at least 1.5 times the font size;
 
Replace this bullet by "Sufficient line height (line spacing) in proportion to the font size" or specify that this particular condition applies to English and the like only.

> Spacing following paragraphs to at least 2 times the font size;

Replace this bullet by "Sufficient spacing after paragraphs in proportion to the font size" or specify that this particular condition applies to English and the like only.

> Letter spacing (tracking) to at least 0.12 times the font size;

Replace this bullet by "Sufficient letter spacing (tracking) in proportion to the font size" or specify that this particular condition applies to English and the like only.

> Word spacing to at least 0.16 times the font size.

Replace this phrase by "Sufficient word spacing in proportion to the font size" or specify that this particular condition applies to English and the like only.

<hr>

There are other suggestions in the  [recent issue](https://github.com/w3c/i18n-discuss/issues/46), and many more in the [long-ish list on the i18n repo](https://github.com/w3c/i18n-activity/issues?q=label%3Awg%3Aag%20sort%3Acreated-asc).

We need to work through issues such as:
- Can we use a word like "sufficient" in normative text, and link to various other standards in informative documents. 
- Can we just except non-latin languages?

I anticipate multiple PRs on this topic, but this issue is to track these potential updates.

**Comment by yatil** [comment]
This would make something that is easily pass/failable in the languages where it applies (I can speak for English and German, where these are sensible sizing options for disabled people) and make it basically untestable (what is “sufficient” in which language?)

**Comment by murata2makoto** [comment]
If the goal is to completely avoid the risk of an unfair recommendation, one approach would be to create a separate specification that applies only to Western languages. This would ensure that the guidance remains meaningful within the linguistic contexts where it is relevant.

On the other hand, if maintaining compatibility with the current WCAG text is a priority, then all provisions that only apply to Western language text should be explicitly marked as such. This would provide clarity on the scope of applicability and prevent misinterpretation in languages where these provisions do not hold.

**Comment by aphillips** [comment]
I was [actioned](https://github.com/w3c/i18n-actions/issues/184) by I18N to respond to this issue. We discussed this in our 2025-08-28 [teleconferece](https://www.w3.org/2025/08/28-i18n-minutes.html#c887) as part of a larger discussion of adding a specific glossary term. 

In general, I18N thinks we need to establish/restore a higher level of interaction and/or earlier review of language-related materials.  We remain concerned about the table-based approach to per-language/per-locale requirements. In our discussion (which was not well-minuted), we noted that orthographic similarity is problematic. It might be difficult for users to determine how their language relates to the exemplar languages. There will be specific languages or writing systems that are similar in some respects to other languages but have divergent needs in others (one of the Unicode [XKCD's](https://xkcd.com/1726/) is obligatory here). 

We recognize that there is a coordination group and that our WG's participation has been incomplete. Open to discussing what approach would help here.

**Comment by shawna-slh** [comment]
Input in GitHub issue for TPAC 2025 breakout : https://github.com/w3c/tpac2025-breakouts/issues/38#issuecomment-3506486553 and following comments
